csharp/ql/src/Likely Bugs/ThreadUnSafeICryptoTransformBad.cs

```diff
@@ -0,0 +1,41 @@
+internal class TokenCacheThreadUnsafeICryptoTransformDemoFixed
```

Comment:
This file should be named `ThreadUnSafeICryptoTransformGood.cs`.

Reply:
Fixed it.